.github/workflows/cifuzz.yml

```
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
```

Let's activate this when merging to master.
```suggestion
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

---

You mean something like this? [build_and_test.yml#L2-L13](https://github.com/cloudflare/roughtime/blob/master/.github/workflows/build_and_test.yml#L2-L13)

I know what this is doing, but this looks very complicated, to be honest.